apps/modernization-ui/src/address/display/displayAddress.tsx

```diff
@@ -35,4 +36,13 @@ const displayAddress = (address: DisplayableAddress): JSX.Element => {
     );
 };
 
-export { displayAddress, displayAddressText };
+/** Returns the type and use as combined text. Example: "Dormitory/Home"
+ * @param {DisplayableAddress} address - The address object with type and use fields.
+ * @return {string} The result string.
+ */
+const displayAddressTypeUse = ({ type, use }: DisplayableAddress): string => {
```

Comment:
nit (non-blocking):  Adding `displayAddressTypeUse` here is lowering the cohesion of this module.  It's more closely aligned with a Patient's Address than the display of an address since it's displaying the type/use alone without an address.  

Would it make more sense to have this function live within `apps/patient/data/address` since it deals with displaying an element of a patient's address that's common among the places where the patient's address is displayed?

Reply:
It seemed like a more appropriate place as it is a common util concerning the display of a component of an address, but I'll move it.

Reply:
@adamloup-enquizit code updated, please review again